proposals/wac-ucr/index.bs

```diff
@@ -911,6 +911,21 @@ write access. Alice grants *PerformChart* read only access to all the projects
 that she can access.
 
 
+### Application discovering access privileges {#uc-client-discover-access-privileges}
+
+Guinan uses an application to author and publish documents. The application
+adapts its UI to the access privileges in order to distinguish between allowed
+(actionable) and disallowed features for different permission groups. The
+application can make use of information to assist Guinan based on what it is
+allowed to perform, the public, as well as any other group.
+
+For example, for user level permission, if the application is not granted
+[=write access=] on the resource that Guinan is currently editing, the UI can
+disable the "Save" button in the menu. Guinan also wants to know if the
+content they are updating is granted [=read access=] to the public -
+consequently, eg. if it can be liked, bookmarked, archived by everyone.
```

Comment:
This could probably benefit from further rewording, but I think this is a significant improvement on the current text.
```suggestion
public is granted [=read access=] on the content they are updating, and
thus if it can be, for example, liked, bookmarked, archived by everyone.
```